content/hi/docs/tasks/tools/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "उपकरण स्थापित करें"
+description: अपने कंप्यूटर पर कुबेरनेट्स टूल सेट करें.
+weight: 10
+no_list: true
+---
+
+## कुबेक्टल
+
+<!-- overview -->
+कुबेरनेट्स कमांड-लाइन टूल, [कुबेक्टल](/docs/reference/kubectl/kubectl/), 
+आपको कुबेरनेट्स क्लस्टर पर कमांड चलाने की अनुमति देता है।
+
+आप एप्लिकेशन को डिप्लॉय करने, क्लस्टर संसाधनों का निरीक्षण 
+और प्रबंधन करने और लॉग देखने के लिए कुबेक्टल का उपयोग कर सकते हैं।
+
+कुबेक्टल संचालन की पूरी सूची सहित अधिक जानकारी के लिए, देखें 
+[`कुबेक्टल` संदर्भ प्रलेखन](/docs/reference/kubectl/).
+
+कुबेक्टल विभिन्न प्रकार के लिनक्स प्लेटफॉर्म, मैकओएस और विंडोज पर इंस्टॉल करने योग्य है।
+नीचे अपना उपयुक्त ऑपरेटिंग सिस्टम खोजें।
+
+- [कुबेक्टल लिनक्स पर इंस्टॉल करने](/docs/tasks/tools/install-kubectl-linux)
+- [कुबेक्टल लिनक्स पर मैकओएस करने](/docs/tasks/tools/install-kubectl-macos)
+- [कुबेक्टल लिनक्स पर विंडोज करने](/docs/tasks/tools/install-kubectl-windows)
+
+## काइंड (`Kind`)
+
+[`काइंड`](https://kind.sigs.k8s.io/docs/) आपको अपने कंप्यूटर पर कुबेरनेट चलाने देता है।
+इस उपकरण के लिए आवश्यक है कि आपके पास 
+[डॉकर](https://docs.docker.com/get-docker/) इंस्टॉल और कॉन्फ़िगर किया गया हो।
+
+काइंड [क्विक स्टार्ट](https://kind.sigs.k8s.io/docs/user/quick-start/) 
+पृष्ठ आपको दिखाता है कि काइंड चलाने के लिए आपको क्या करने की आवश्यकता है।
+
+<a class="btn btn-primary" href="https://kind.sigs.k8s.io/docs/user/quick-start/" role="button" aria-label="View kind Quick Start Guide">काइंड क्विक स्टार्ट गाइड देखें</a>
+
+## मिनीक्यूब
```

Comment:
```suggestion
## minikube
```

It's OK to provide a hint about how to say this word. We do need to make it clear what its real name is though.